test/ex_member_page_test.rb

```diff
@@ -1,20 +1,27 @@
 # frozen_string_literal: true
 require_relative './test_helper'
 require_relative '../lib/ex_member_page.rb'
+require_relative '../lib/decorators/formatted_dates'
+require_relative '../lib/electoral_history'
 
 describe MemberPage do
-  around { |test| VCR.use_cassette('GeorgeVella', &test) }
+  around { |test| VCR.use_cassette('JoeCassar', &test) }
 
   subject do
-    url = 'http://www.parlament.mt/vella-george'
+    url = 'http://www.parlament.mt/joe-cassar'
     ExMemberPage.new(response: Scraped::Request.new(url: url).response)
   end
 
   it 'should have a name field with the name of the member' do
-    subject.name.must_equal 'George Vella'
+    subject.name.must_equal 'Dr Joe Cassar'
   end
 
   it 'should respond to electoral_history' do
     subject.must_respond_to :electoral_history
   end
+
+  it 'should have an end date for the twelfth term' do
+    subject.electoral_history['Twelfth Parliament']['Resignation from Parliament']
+           .must_equal '2015-11-03'
+  end
```

Comment:
this isn't really testing very much of what the PR claims to be adding. Let's talk in person about a better way of testing things like this.